libs/wcm/utils/connectionCreator.js

```diff
@@ -0,0 +1,20 @@
+import SignClient from '@walletconnect/sign-client';
+import pkg from '../../../package.json';
+
+// eslint-disable-next-line import/no-mutable-exports
+export let client;
+
+export async function createSignClient() {
+  client = await SignClient.init({
+    projectId: process.env.PROJECT_ID,
+    relayUrl: process.env.RELAY_URL,
+    metadata: {
+      name: pkg.name,
+      description: pkg.description,
+      url: pkg.homepage,
+      icons: ['https://lisk.com/documentation/_/img/lisk-symbol.svg'],
```

Comment:
Maybe add a TODO to replace this URL with the one we provide via service metadata URL

Reply:
This is the Logo of Lisk Desktop and will remain unchanged. Do you mean we plan to serve Lisk logo from Service? In that case, I'll add a todo as you requested.

Reply:
Yes, the logos will now come from service (app and token logos)

Reply:
Cool. I'll add a todo and create a ticket then.